bin/nlg-eval

```diff
@@ -9,10 +9,15 @@ import stat
 import sys
 import time
 from zipfile import ZipFile
+from xdg import XDG_CONFIG_HOME, XDG_CACHE_HOME
```

Comment:
We've been pretty good in these files about using a nice ordering (standard lib, 3rd party, ours). Would you mind moving this import to match https://www.python.org/dev/peps/pep-0008/#imports ?

Reply:
done